clippy_lints/Cargo.toml

```diff
@@ -10,6 +10,7 @@ edition = "2021"
 
 [dependencies]
 cargo_metadata = "0.14"
+clippy_dev = { path = "../clippy_dev", optional = true }
```

Comment:
Just noticed during the sync: depending on `clippy_dev` feels really wrong. clippy_dev isn't intended to be used as a library, but a standalone binary for clippy dev tools. Nothing should depend on `clippy_dev` IMO.

Reply:
Ups yeah your right, slipped under my radar😅